### PR TITLE
[UI] Connections

### PR DESCRIPTION
**Pull request by senali-d**
**Notes for Reviewers**

https://github.com/meshery/meshery/assets/52546856/ceaa2be0-443d-45bc-91d1-506132749bff

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
Hii @senali-d 
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

File: ui/components/connections/index.js [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+    "& .MuiChip-label" : {
+      color : "#477E96 !important",
+    },
+    background : "#477E9615 !important"
+  },
+  discovered : {
+    "& .MuiChip-label" : {
+      color : "#EBC017",
+    },
+    background : "#EBC01715 !important"
```
Comment (leecalcote) [inline ui/components/connections/index.js:89]: theme reference?

**Comment by senali-d** [comment]
@ritiksaxena124 Do you have icon for connections
conn1

**Comment by ritiksaxena124** [comment]
we can use the same connection icon that we are using in meshery cloud for connections

connections (1)

**Comment by UsmanDev09** [comment]
LGTM

**Pull request merged**